Gruntfile.coffee

```diff
@@ -255,8 +255,8 @@ module.exports = (grunt) ->
   grunt.registerMultiTask "build_test_runner_file", "Creates a test runner file.", ->
```

Comment:
I tried to execute it and it fails ( `tests/support/runner.html.tmpl` does not exist ). I doubt anyone is using it.